src/bot.py

```diff
@@ -37,9 +37,7 @@
 
 mininterval = config['mininterval']
 maxinterval = config['maxinterval']
-hours_to_wait = randint(mininterval, maxinterval) * 24
-hour_in_minutes = 3600
-cooldown = hours_to_wait * hour_in_minutes
+cooldown = randint(mininterval, maxinterval) * 24 * 3600

```

Comment:
Also, make sure to add a simple comment explaining the reasoning behind those multiplications to any potential future onlookers. :D

Reply:
brb